redirects-list.json

```diff
@@ -28,11 +28,17 @@
   "^/doc/tutorials(/.*)?                                                                  /doc/start",
 
   "^/doc/use-cases/data-and-model-files-versioning/?$                                     /doc/use-cases/versioning-data-and-model-files",
-  "^/doc/user-guide/dvc-file-format$                                                      /doc/user-guide/dvc-files-and-directories",
-  "^/doc/user-guide/dvc-files-and-directories$                                            /doc/user-guide/dvc-files",
+  "^/doc/user-guide/dvc-file-format$                                                      /doc/user-guide/project-structure",
+  "^/doc/user-guide/dvc-files-and-directories$                                            /doc/user-guide/project-structure",
+  "^/doc/user-guide/dvc-files$                                                            /doc/user-guide/project-structure",
   "^/doc/user-guide/updating-tracked-files$                                               /doc/user-guide/how-to/update-tracked-data",
   "^/doc/user-guide/how-to/update-tracked-files$                                          /doc/user-guide/how-to/update-tracked-data",
   "^/doc/user-guide/merge-conflicts$                                                      /doc/user-guide/how-to/merge-conflicts",
+  "^/doc/user-guide/dvc-files/dvc-yaml$                                                   /doc/user-guide/project-structure/pipelines-files",
```

Comment:
no need for this redirect - we've released it not long ago, let's not complicate this

Reply:
Actually this is probably necessary at least temporarily, because https://dvc.org/doc/user-guide/dvc-files already received the rank and traffic from the previous URLs (currently [indexed](https://search.google.com/u/1/search-console/inspect?resource_id=sc-domain%3Advc.org&id=X4AehNlF5NqXR8XBNrJVUA) by Google):

![image](https://user-images.githubusercontent.com/1477535/105793117-2fdc8100-5f4e-11eb-9baf-5c9315a1ec05.png)

We want the traffic and rank to transfer to the new URL ([/doc/user-guide/project-structure](https://dvc-landing-2-0-pipelin-rttezd.herokuapp.com/doc/user-guide/project-structure)) I think.

Reply:
But what we can do now is remove the previous 2 redirect lines above (those URLs are no longer indexed on Google) to simplify the redirects list.

> It could break a few links from old web pages or blog posts outside our domain though.